docs/docsite/rst/network/user_guide/platform_cnos.rst

```diff
@@ -0,0 +1,69 @@
+.. _cnos_platform_options:
+
+***************************************
+CNOS Platform Options
+***************************************
+
+CNOS supports Enable Mode (Privilege Escalation). This page offers details on how to use Enable Mode on CNOS in Ansible 2.7. 
```

Comment:
@gundalow @privateip Is there any reason we should keep a specific version in these platform guides?
@amuraleedhar This version says 2.7 but the settings by platform table entries say this is supported as of 2.5. Let's see what the other two folks say, but I think we can remove the specific version in this intro sentence.

Reply:
Removing 2.7 now. Let it be generic. For specific reasons users may refer to the table.

Reply:
Removing 2.7 now